main.go

```diff
@@ -45,10 +22,8 @@ func main() {
 		log.SetLevel(log.DebugLevel)
 	}
 
-	creds, err := getCredentials()
-	if err != nil {
-		log.WithError(err).Fatal("unable to get credentials")
-	}
+	cfg := defaults.Get().Config
+	creds := cfg.Credentials
 
 	signer := v4.NewSigner(creds)
```

Comment:
Since we aren't checking for an error anymore and we aren't reusing these vars we could just make this ```signer := v4.NewSigner(defaults.Get().Config.Credentials)```

Can you update the readme to explain how credentials are found? (maybe with a link to the golang sdk docs for defaults)